server.js

```diff
@@ -1,32 +1,146 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+import netflixData from "./data/netflix-titles.json";
+import dotenv from 'dotenv';
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
+dotenv.config();
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
+
+// from starter code: const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || "mongodb://127.0.0.1:27017/project-mongo";
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true, dbName: "project-mongo" })
+  .then(() => console.log('MongoDB connected successfully'))
+  .catch((err) => console.error(`MongoDB connection error: ${err}`));
 mongoose.Promise = Promise;
 
 // Defines the port the app will run on. Defaults to 8080, but can be overridden
 // when starting the server. Example command to overwrite PORT env variable value:
 // PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
+// Will create a list of all endpoints in our API, first do: npm install express-list-endpoints
+const listEndpoints = require('express-list-endpoints');
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+
+// In MongoDB, a schema defines the structure of a document, including the field names, data types, and any constraints or validations. 
+// It serves as a blueprint for how the data should be organized and is used to ensure consistency in the data
+// Schema is basically a (mongoose) skeleton
```

Comment:
This is good information, especially for someone new to mongoose and schemas. But I would consider using commenting that helps the reader to understand what happens in the code more specifically. Like: 'This schema does this and that'.
Just an opinion :)